<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AR600E - интерфейс управления</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,10" columnstretch="0,10,0">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="PowerWidgetLayout"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="RecordWidgetLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
       </layout>
      </item>
      <item row="0" column="2" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_5" rowstretch="2,8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="MoveControlLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="MotorControlLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabDriver">
         <attribute name="title">
          <string>Двигатели</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="MotorTableLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSensor">
         <attribute name="title">
          <string>Датчики</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="SensorTableLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Список файлов для выполнения</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="MoveFilesWidgetLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenConfigFile"/>
    <addaction name="actionSaveConfigFile"/>
    <addaction name="actionOpenCorrectionFile"/>
    <addaction name="action_Open_DRIVEMAT"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Соединение</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionOpenConnectConfig"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Питание</string>
    </property>
    <addaction name="actionOn"/>
    <addaction name="actionOff"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="MainToolBar">
   <property name="windowTitle">
    <string>Панель настроек</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenConfigFile">
   <property name="text">
    <string>Открыть файл настроек</string>
   </property>
  </action>
  <action name="actionSaveConfigFile">
   <property name="text">
    <string>Сохранить файл настроек</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>Включить все</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Выключить все</string>
   </property>
  </action>
  <action name="actionOpenConnectConfig">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionOpenCorrectionFile">
   <property name="text">
    <string>Открыть файл обр. связи</string>
   </property>
  </action>
  <action name="action_Open_DRIVEMAT">
   <property name="text">
    <string>Открыть файл DRIVEMAT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
